.github/workflows/test-plan-item-validator.yml

```diff
@@ -0,0 +1,24 @@
+name: Test Plan Item Validator
+on:
+  issues:
+    types: [edited, labeled]
+
```

Comment:
We locked down actions' access to our GitHub token some time ago. Depending on what this action does you might need to specify `permissions` for it to work: https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token e.g. https://github.com/microsoft/vscode-jupyter/blob/bec563336a7cc7eed837835918fbd282b58469f1/.github/workflows/codeql-analysis.yml#L14-L15

Reply:
Thanks, added scopes